.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+**What type of PR is this?**
+<!--
+Add one of the following kinds:
+/feat: new feature
+/bugs: fix bug
```

Comment:
https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
IMO, the `/xxx` style is used in k8s communities and prow style to add label automatically. Like:

/kind bug
/kind feature
/kind refactor

etc.